sdks/sdk-core/src/chains.ts

```diff
@@ -22,6 +22,7 @@ export enum ChainId {
   ZORA_SEPOLIA = 999999999,
   ROOTSTOCK = 30,
   BLAST = 81457,
+  ZKSYNC = 77,
```

Comment:
whered you get 77? I see 324 https://chainlist.org/?search=zksync

Reply:
i also see 324

Reply:
Sorry github copilot autocomplete. Usually it gets the chain id right, but it got wrong, and I didn't double check.